src/test/java/org/broadinstitute/hellbender/tools/reference/CheckReferenceCompatibilityIntegrationTest.java

```diff
@@ -0,0 +1,205 @@
+        final File ref3 = new File(COMPARE_REFERENCES_TEST_FILES_DIRECTORY + "hg19mini_chr2snp.fasta");
+        final File dict = new File(getToolTestDataDir() + "reads_data_source_test1_withmd5s_missingchr1.bam");
+        final File output = createTempFile("testReferenceCompatibilityMultipleReferencesWithMD5s", ".table");
+        final File expectedOutput = new File(getToolTestDataDir(), "expected.testReferenceCompatibilityMultipleReferencesBAMWithMD5s.table");
+
+        final String[] args = new String[] {"-refcomp", ref1.getAbsolutePath(), "-refcomp", ref2.getAbsolutePath(),
+                "-refcomp", ref3.getAbsolutePath(), "-I", dict.getAbsolutePath(), "-O", output.getAbsolutePath()};
+        runCommandLine(args);
+
+        IntegrationTestSpec.assertEqualTextFiles(output, expectedOutput);
+    }
+
+    // Note that hg19mini_chr2snp.fasta is NOT a valid reference, but tool renders them a match due to compatibility by name and length.
+    // Included to demonstrate that this is the best the tool can do without MD5s present
+    @Test
+    public void testReferenceCompatibilityMultipleReferencesBAMWithoutMD5s() throws IOException {
+        final File ref1 = new File(COMPARE_REFERENCES_TEST_FILES_DIRECTORY + "hg19mini.fasta");
+        final File ref2 = new File(COMPARE_REFERENCES_TEST_FILES_DIRECTORY + "hg19mini_1renamed.fasta");
+        final File ref3 = new File(COMPARE_REFERENCES_TEST_FILES_DIRECTORY + "hg19mini_chr2snp.fasta");
+        final File dict = new File(getToolTestDataDir() + "reads_data_source_test1_withoutmd5s.bam");
+        final File output = createTempFile("testReferenceCompatibilityMultipleReferencesWithoutMD5s", ".table");
+        final File expectedOutput = new File(getToolTestDataDir(), "expected.testReferenceCompatibilityMultipleReferencesBAMWithoutMD5s.table");
+
+        final String[] args = new String[] {"-refcomp", ref1.getAbsolutePath(), "-refcomp", ref2.getAbsolutePath(),
+                "-refcomp", ref3.getAbsolutePath(), "-I", dict.getAbsolutePath(), "-O", output.getAbsolutePath()};
+        runCommandLine(args);
+
+        IntegrationTestSpec.assertEqualTextFiles(output, expectedOutput);
+    }
+
+    // References are not compatible since one has SNP on chr 2, but tool renders them seemingly a match
+    // Included to demonstrate that this is the best the tool can do without MD5s present
+    @Test
+    public void testReferenceCompatibilityWithoutMD5sMismatch() throws IOException {
+        final File ref1 = new File(COMPARE_REFERENCES_TEST_FILES_DIRECTORY + "hg19mini_chr2snp.fasta");
+        final File dict = new File(getToolTestDataDir() + "reads_data_source_test1_withoutmd5s.bam");
+
+        final String[] args = new String[] {"-refcomp", ref1.getAbsolutePath() , "-I", dict.getAbsolutePath()};
+        runCommandLine(args);
+    }
+
+    // TODO: compatibility based on MD5 faulty since MD5s not in sequence dictionary (see ticket #730 "VCFHeader drops sequence dictionary attributes")
```

Comment:
Since #730 is in another repository (htsjdk) make this the full link since this is going to be confusing.